readthedocs/api/v3/views.py

```diff
@@ -614,21 +619,47 @@ def get_queryset(self):
         )
 
 
-class OrganizationsViewSet(
+class OrganizationsViewSetBase(
     APIv3Settings,
     GenericViewSet,
 ):
     # NOTE: this viewset is only useful for nested URLs required for notifications:
     # /api/v3/organizations/<slug>/notifications/
     # However, accessing to /api/v3/organizations/ or /api/v3/organizations/<slug>/ will return 404.
     # We can implement these endpoints when we need them, tho.
+    # Also note that Read the Docs for Business expose this endpoint already.
 
     model = Organization
     serializer_class = OrganizationSerializer
     queryset = Organization.objects.none()
     permission_classes = (IsAuthenticated,)
 
 
+class OrganizationsViewSet(SettingsOverrideObject):
+    _default_class = OrganizationsViewSetBase
```

Comment:
Any reason why we are writing this code on .com instead of having it here? .org doesn't have organizations, so there isn't a difference between implementations.

Reply:
Mainly, because of permissions. Right now, this endpoint is _disabled_ on community, but it does return results on commercial. I think we can eventually refactor this code to move it out from .com and adapt it to be compatible on both platform.

For now, I want to fix the endpoints that I deleted by mistake while working on notifications, make the corresponding tests to pass, and move forward with the deploy.